nix-tools-default.nix

```diff
@@ -0,0 +1,45 @@
+commonLib: # the iohk-nix commonLib, that provides access to the pinned packages
+pkgs-path: # the path to th local pkgs.nix file for nix-tools that imports all
```

Comment:
the

Reply:
Suggest putting "haskell" in the name of this variable somewhere to avoid confusing it with the `pkgs` that means nixpkgs.

Reply:
I think `nix-tools` might even be better than `haskell`, as this is the `pkgs.nix` file as generated by the `nix-tools` tooling.